solutions/swb-reference/integration-tests/tests/isolated/environments/launch.test.ts

```diff
@@ -60,14 +62,49 @@ describe('environments launch negative tests', () => {
     });
   });
 
+  describe('Project admin and researcher not associated with project', () => {
+    test('Unable to launch - project admin', async () => {
+      try {
+        // await pa1Session.resources.projects
```

Comment:
Using code snippet from 68 - 71 throws error `CreateEnvironments requires a parent Project resource.`

Reply:
Ah the problem is line 13 asks for only one project, so project2Id is coming in as undefined, which sends an undefined value up when constructing the url.